### ISSUE TITLE
Accessibility: Object Status: Text alternative/tooltip is not provided for object status icons, defining the purpose of icons.

### ISSUE THREAD

**Issue by puru-hk** [SAP/fundamental-ngx#3444]
Description: Text alternative/tooltip is not provided for object status icons, defining the purpose of icons.

Expected: Text alternative should be provided for each & every non-text UI, describing the purpose of UI’s. 

Attached screenshot:

![image](https://user-images.githubusercontent.com/32538291/94520322-ae980a80-0249-11eb-971b-bde6da1ee870.png)

**Comment by Lokanathan-k** [comment]
fixed as part of the PR https://github.com/SAP/fundamental-ngx/pull/4176
